libpod/container.go

```diff
@@ -365,6 +366,25 @@ type ContainerConfig struct {
 	Systemd bool `json:"systemd"`
 }
 
+// AttachStreams contains streams that will be attached to the container
```

Comment:
Throw this in container_api.go right above the Attach() function

Reply:
done